tasks/set_facts.yml

```diff
@@ -0,0 +1,36 @@
+- name: Get correct path for xl if needed
+  shell: which xl
```

Comment:
Also install `which`, it's in debianutils iirc

Reply:
done